constrain/Classes/Constraints.swift

```diff
@@ -4,43 +4,22 @@
 
 import UIKit
 
-public enum ConstraintIdentifier: String {
-    case top = ".top"
-    case leading = ".leading"
-    case trailing = ".trailing"
-    case bottom = ".bottom"
-    case topSafe = ".topSafe"
-    case bottomSafe = ".bottomSafe"
-    case height = ".height"
-    case width = ".width"
-    case centerX = ".centerX"
-    case centerY = ".centerY"
-    case aspectRatio = ".aspectRatio"
-
-    // Not actually constraints:
-    case cornerRadius = ".cornerRadius"
-    case size = ".size" // size combines .width and .height
-}
-
 public typealias Relationship = NSLayoutConstraint.Relation
 
-public typealias ConstraintSet = [NSLayoutConstraint]
-
 /// Constraints is a container of layout constraints for a UIView. It has convenience methods for creating and retrieving constraints.
-/// For improved readability constraint-creation methods can be chained.
+/// For improved readability, all methods can be chained.
 /// Constraints are active on creation
 /// Most methods allow the omission of an anchor or view to be passed in.
 /// The superview is used when no view is passed in and the superview's respective anchor is used when the anchor is omitted.
 public class Constraints {
     
     internal weak var view: UIView?
     internal var viewName: String
-    private var constraints: [ConstraintIdentifier: NSLayoutConstraint] = [:]
+    internal var constraints: [ConstraintIdentifier: NSLayoutConstraint] = [:]
+    internal var allConstraints: ConstraintSet = []
```

Comment:
Haven't really thought about this before and out of scope for this PR but should we be holding strong references to `NSLayoutConstraint`s? I imagine that `UIView`s hold a strong reference and if we ever held a reference to `Constraints` that would introduce a memory cycle if I'm not mistaken.

Reply:
I think that would only be the case if the constraint held a strong reference to the view. So I think the ownership is 

```
view or vc ------> NSLayoutconstraint
   |                           ^
    ----------> constrain -----
```

which should be fine, right?